docs/conf/eu/2016/news/thanks-for-coming.md

```diff
@@ -0,0 +1,64 @@
+:template: 2016/eu.html
+
+Thanks for coming to Write the Docs Europe 2016
+===============================================
+
+Hi everyone,
+
+Thanks for attending our third European Write the Docs conference in Prague! We're so grateful to our sponsors, speakers, volunteers, and attendees for making the conference possible. 
+
+We hope you had a wonderful time, and that you are as excited as we are about the beautiful community that is forming. Below we've included all of the resources you might need for staying in touch, like the Write the Docs Slack. We also have a short anonymous survey for you to fill out, so we can make the conference even better next year.
```

Comment:
"like the WTD slack" doesn't feel like it's necessary here, we can just keep the sentence short